packages/mirrorful/cli/helpers/openBrowser.ts

```diff
@@ -0,0 +1,12 @@
+import open from 'open'
+type props = { url: string; port: number }
+export default async function openBrowser({ url, port }: props) {
+  try {
+    console.log(`Opening up browser to ${url}:${port}`)
+    const res = await open(`${url}:${port}`)
+
+    console.log('🚀 ~ file: openBrowser.ts:8 ~ openBrowser ~ res:', res)
```

Comment:
```suggestion
    console.log('🚀 Opening Mirrorful in your browser! ', res)
```

Reply:
I think maybe we should just remove the result variable from the log as it might bloat up the console.